FlowersStore/Scenes/FlowersList/FlowerItemCell.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  FlowerItemCell.swift
+//  FlowersStore
+//
+//  Created by Renato on 03/11/21.
+//
+
+import Foundation
+import UIKit
+
+struct Koe {
+    let name: String
+}
+
+protocol FlowerItemCellDelegate: AnyObject {
+    func didTapFavoriteButton(_ cell: FlowerItemCell, flower: Flower)
+}
+
+final class FlowerItemCell: UICollectionViewCell {
+    // MARK: - Properties
+    private weak var delegate: FlowerItemCellDelegate?
+    private var flower: Flower?
+    private var imageDownloadProtocol: ImageDownloadProtocol?
+
+    var teste: String?
+    
+    var teste_2: String?
```

Comment:
<!--
  0 failure: 
  1 warning:  Property 'teste_2...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowerItemCell.swift;
  Line: 27;
-->

- :warning: Property 'teste_2' is unused